processing/tests/gazette/locations/test_ba_feira_de_santana.py

```diff
@@ -0,0 +1,193 @@
+import datetime
+import json
+import pytest
+
+from gazette.locations.ba_feira_de_santana import BaFeiraDeSantana
```

Comment:
I wasn't able to run the tests because the imports weren't found. By the way: what's the right way of running the tests here? Just hitting `pytest`? Would be nice to add it to `test` on Makefile.